### ISSUE TITLE
AVT 3 - Carbon X Progress Indicator Component can not be screen reader tested

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1827]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description
AVT 3 The progress indicator cannot be tested without a complete example so the user can navigate through the steps and confirm there is a meaningful sequence and the warning indicator is read at the appropriate time. 
[See WCAG SC 1.3.2](http://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)
 [See WCAG Guidelines 3.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error.html)

**Comment by dakahn** [comment]
Expected/suggested implementation as well as screen reader/a11y automated testing results are coming soon.
